docs/architecture/specs/examples/did/v0.4/ddo-example.json

```diff
@@ -0,0 +1,334 @@
+				"tag": "v0.1",
+				"createdAt": "2016-02-08T16:02:20Z",
+				"checksum": "4329042309a02394032"
+			},
+			{
+				"_id": "ccacd",
+				"tag": "v0.2",
+				"createdAt": "2018-02-08T16:02:20Z",
+				"checksum": "0000010992349"
+			}
+		],
+		"networks": [{
+				"chainId": 1
+			},
+			{
+				"chainId": 23
+			},
+			{
+				"chainId": 8001
+			}
+		]
+	},
+	"service": [{
+			"index": 0,
+			"serviceEndpoint": "http://localhost:5000/api/v1/metadata/assets/ddo/{did}",
+			"type": "metadata",
+			"attributes": {
+				"encryptedFiles": "0x2e48ceefcca7abb024f90c87c676fce8f7913f889605a349c08c0c4a822c69ad651e122cc81db4fbb52938ac627786491514f37a2ebfd04fd98ec726f1d9061ed52f13fde132222af34d9af8ec358429cf45fc669f81a607185cb9a8150df3cbb2b4e3e382fb16429be228ddd920f061b78dd54701025fac8aab976239fb31a5b60a57393e96a338324c5ac8a5600a1247339c4835533cecdb5b53caf6b6f9d6478b579b7426f650a4154a20d18a9d49f509770af62647a57fc174741b47af3c8beeaaa76bee276cce8fba1f3fec0e1c",
+				"main": {
+					"author": "Met Office",
+					"dateCreated": "2019-02-08T08:13:49Z",
+					"datePublished": "2019-02-08T08:13:49Z",
+					"files": [{
+						"url": "https://raw.githubusercontent.com/tbertinmahieux/MSongsDB/master/Tasks_Demos/CoverSongs/shs_dataset_test.txt",
+						"index": 0,
+						"checksum": "efb2c764274b745f5fc37f97c6b0e761",
+						"contentLength": "4535431",
+						"contentType": "text/csv",
+						"encoding": "UTF-8",
+						"compression": "zip"
+					}],
+					"license": "CC-BY",
+					"name": "UK Weather information 2011",
+					"type": "dataset"
+				},
+
+				"additionalInformation": {
+
+					"description": "Weather information of UK including temperature and humidity",
+					"tags": ["weather", "uk", "2011", "temperature", "humidity"],
+					"workExample": "423432fsd,51.509865,-0.118092,2011-01-01T10:55:11+00:00,7.2,68",
+					"copyrightHolder": "Met Office",
+					"links": [{
+							"name": "Sample of Asset Data",
+							"type": "sample",
+							"url": "https://foo.com/sample.csv"
+						},
+						{
+							"name": "Data Format Definition",
+							"type": "format",
+							"url": "https://foo.com/sampl2.csv"
+						}
+					],
+					"inLanguage": "en",
+
+					"updateFrecuency": "yearly",
+					"structuredMarkup": [{
+							"uri": "http://skos.um.es/unescothes/C01194/jsonld",
+							"mediaType": "application/ld+json"
+						},
+						{
+							"uri": "http://skos.um.es/unescothes/C01194/turtle",
+							"mediaType": "text/turtle"
+						}
+					]
+				},
+
+				"curation": {
+					"numVotes": 123,
+					"rating": 0.0,
+					"schema": "Binary Votting",
+					"isListed": true
+				}
+			}
+		},
+		{
+			"type": "access",
+			"index": 1,
+			"serviceEndpoint": "http://localhost:8030/api/v1/gateway/services/consume",
+			"templateId": "",
+			"attributes": {
+				"main": {
+					"name": "dataAssetAccessServiceAgreement",
+					"creator": "",
+					"datePublished": "2019-02-08T08:13:49Z"
+				},
+				"additionalInformation": {
+					"description": ""
+				},
```

Comment:
I will remove this field also since it's not being used and its optional, just to keep the DDOs smaller and cleaner

Reply:
`additionalInformation` is a key-value bucket that in most of the cases should be empty or it doesn't exist. Its good to have it in the Spec because different use cases could require to use additional attributes out of the default ontology.